content/2018-09-25-newsletter-12.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "The Embedded Working Group Newsletter - 12"
+date = 2018-09-25
+draft = false
+in_search_index = true
+template = "page.html"
++++
+
+This is the twelfth newsletter of the [Embedded WG] where we highlight new progress, celebrate cool projects, thank the community, and advertise projects that need help!
+
+[Embedded WG]: https://github.com/rust-embedded/wg
+
+Discuss on [internals.rust-lang.org], [on twitter], or [on reddit]!
+
+[internals.rust-lang.org]: #
+[on twitter]: #
+[on reddit]: #
+
+<!-- more -->
+
+If you want to mention something in [the next newsletter], make sure to leave a comment on the issue.
+
+[the next newsletter]: https://github.com/rust-embedded/blog/issues/15
+
+## Highlights
+
+* We now have a category for Embedded on the [users.rust-lang.org] forum! This is a great place to ask larger questions, or to share more details about your projects!
+* You can now build embedded [binaries on stable] with the `1.30-beta` release of Rust! Check it out and let us know if you run in to any issues with your projects!
```

Comment:
This could mention how to best let us know.

Reply:
Adding a link to the /wg issue tracker